rlog.go

```diff
@@ -49,6 +51,23 @@
 //                     higher the RLOG_TRACE_LEVEL value, the more 'chatty' and
 //                     verbose the Trace message output becomes.
 //                     Default: -1
+// * RLOG_CALLER_FILTER:
```

Comment:
I think (slight modification from my earlier comments): RLOG_CALLER_FILTER should be removed, RLOG_TRACE_LEVEL should be kept, but should take on RLOG_CALLER_FILTER functionality. In addition, you should just be able to defined a number, as a general level.